dev/doc/changes.txt

```diff
@@ -51,6 +51,12 @@ In Constrexpr_ops:
   interpreting "(x y z:_)" as "(x:_) (y:_) (z:_)" while the second
   ones were preserving the original sharing of the type.
 
+In Nameops:
+
+  The API has been made more uniform. New combinators added in the
+  "Name" space name. Function "out_name" now fails with IsAnonymous
+  rather than with Failure "Nameops.out_name".
+
```

Comment:
> But, honestly, I'm a bit perplex in what we can achieve in terms of consistent API design. I proposed this change in Nameops as a possible direction to go towards unified naming conventions but as said above the burden coming with this kind of change is worth in my opinion only if made systematic and I'm not able at the current time to evaluate if the team has enough energy to do it systematically. Somehow it depends a lot on you. Assuming a consensus on a DataStructure.operator model, and there seems to be such consensus these days for it, would you be ok, if you have no other priorities, to work at implementing it (e.g. turning iter_glob_constr into GlobConstr.iter, etc., but in a systematic way, which is not necessarily easy, because we shall find quickly cases which go out of the standard situations)?

The only reason why I am not raising the same point is that I am waiting for [this beast](https://github.com/coq/coq/pull/652) to be integrated. Instead of women, these days I dream about merge conflicts.